src/GraphQL/loaders/parseClassTypes.js

```diff
@@ -284,30 +202,6 @@ const load = (
     classGraphQLUpdateType
   );
 
-  const classGraphQLPointerTypeName = `${graphQLClassName}PointerInput`;
```

Comment:
Why are you removing this one?

Reply:
That was a mistake, thanks for noticing it. It was supposed to remove only the Pointer scalar type.